api/scpca_portal/test/models/test_job.py

```diff
@@ -101,12 +102,7 @@ def test_submit_pending(self, mock_batch_submit_job):
                 state=JobStates.PENDING,
                 dataset=DatasetFactory(is_pending=True, is_processing=False),
             )
-        for state in [
-            JobStates.PROCESSING,
-            JobStates.SUCCEEDED,
-            JobStates.FAILED,
-            JobStates.TERMINATED,
-        ]:
+        for state in [JobStates.PROCESSING] + common.FINAL_JOB_STATES:
```

Comment:
Might make sense to just add `common.SUBMITTED_JOB_STATES`

Reply:
That's a good idea to keep the codebase DRY, added!